toml/src/main/java/com/fasterxml/jackson/dataformat/toml/TomlReadFeature.java

```diff
@@ -0,0 +1,41 @@
+package com.fasterxml.jackson.dataformat.toml;
+
+import com.fasterxml.jackson.core.FormatFeature;
+
+/**
+ * Enumeration that defines all togglable features for TOML parsers.
+ */
+public enum TomlReadFeature
+        implements FormatFeature {
+    USE_BIG_INTEGER_FOR_INTS(false),
```

Comment:
Maybe brief javadoc explanation?

Reply:
Still not sure if there's a better way to do this. e.g. the heuristic for BigInteger vs smaller int types that JsonParser uses